spec/v1.0.0/spec.md

```diff
@@ -346,7 +346,16 @@ values, and an explanation why a certain item or set of data is being requested:
   the target schema represents.
 - The [[ref:Input Descriptor Object]] ****MAY**** contain a `purpose` property.
   If present, its value ****MUST**** be a string that describes the purpose for
-  which the [[ref:Claim]]'s data is being requested.
+  which the [[ref:Claim]]'s data is being used for.
```

Comment:
Like juan said in slack, just capturing here for better visibility, this `spec/v1.0.0/spec.md` file is ratified so we cannot edit it any more (I think small things like typos are allowed). Instead can you please update this in the [`spec/spec.md`](https://github.com/decentralized-identity/presentation-exchange/blob/master/spec/spec.md) file? That is the "live" spec that's taking updates :)

Thank you for the PR!!